packages/ui/lib/components/AlbumPreview/index.tsx

```diff
@@ -1,32 +1,54 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import _ from 'lodash';
 import { Button } from 'semantic-ui-react';
 
 import { getThumbnail } from '../..';
+import { Track } from '../../types';
 import ContextPopup from '../ContextPopup';
 import TrackRow from '../TrackRow';
 import artPlaceholder from '../../../resources/media/art_placeholder.png';
 import styles from './styles.scss';
 
-const AlbumPreview = ({
-  album,
-  trackButtons,
+type Artist = {
+  name: string;
+};
+
+type Image = {
+  '#text'?: string;
+};
+
+type AlbumProps = {
```

Comment:
Is it possible to bring this to `types.ts` so it can be used in other components?

Reply:
Sure, will do.